liqd_lunch/frontend/src/ResistanceList.js

```diff
@@ -54,7 +54,7 @@ class  ResistanceList  extends  Component {
                     <span>{c.name}</span>
                     <span>{c.total_resistance}</span>
                   </div>)}
-                  {this.state.nextPageURL == '' &&
+                  {this.state.nextPageURL === '' &&
```

Comment:
I know this wasn't in this PR, but here's a ! missing, right?!

Reply:
Weird, a have seen a "next" button, that is leading nowhere somewhere, but in this PR, I can't find it, so will ignore for now!